maquetado/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="textAndDescription">
+    <p class="title">Titulo del Libro</p>
+    <p class="description subtitle">(género)</p>
+  </div>
+  <div class="textAndDescription">
+    <p class="titleDescription">Autor del libro:</p>
+    <p class="description">Nombre del autor del libro</p>
+  </div>
+  <div class="textAndDescription">
+    <h1 class="titleDescription">Editorial:</h1>
+    <p class="description">Nombre de la editorial</p>
+  </div>
+  <div class="textAndDescription">
+    <p class="titleDescription">Año de publicación:</p>
+    <p class="description">Año de publicación</p>
+  </div>
+</body>
+
+</html>
```

Comment:
We should cheek why the eslint is fixing this every time you save your file :eyes: